README.md

```diff
@@ -21,24 +21,25 @@ A.1 [How to change kernel configurations](#kernel)
 
 If you are new to Yocto, it is recommended to read the [Yocto Overview and Concepts Manual](https://docs.yoctoproject.org/overview-manual/index.html). To get a quick introduction to Yocto, this [Software Overview](https://www.yoctoproject.org/software-overview) might be helpful. For further documentation on the Yocto Project, including information about dealing with BSP layers and working with the Yocto Project's build system **BitBake**, check the [Yocto Project Documentation](https://docs.yoctoproject.org/).
 
 ## Background <a name="background"></a>
 
 ### Layers <a name="layers"></a>
 
-As the Yocto Project is based on the concept of [layers](https://docs.yoctoproject.org/dev-manual/common-tasks.html#understanding-and-creating-layers), the following table lists all the layers used to create a basic distribution based on chargebyte’s charge control platforms Tarragon and EVAcharge SE.
+As the Yocto Project is based on the concept of [layers](https://docs.yoctoproject.org/dev-manual/common-tasks.html#understanding-and-creating-layers), the following table lists all the layers used to create a basic distribution based on chargebyte’s Linux platforms.
 
 | Layer | Description | Repository |
 |--|--|--|
-| meta-chargebyte | BSP layer for Tarragon & EVAcharge SE | https://github.com/chargebyte/meta-chargebyte |
-| meta-chargebyte-distro | Distribution adaptations layer | https://github.com/chargebyte/meta-chargebyte-distro |
+| poky | Build tool and metadata included in a reference distribution | https://git.yoctoproject.org/poky |
 | meta-freescale | Layer containing NXP hardware support metadata | https://git.yoctoproject.org/cgit/cgit.cgi/meta-freescale |
 | meta-openembedded | Collection of layers to supplement OE-Core with additional packages | https://github.com/openembedded/meta-openembedded |
+| meta-security | Collection of security related layers, here used for e.g. TPM support | https://git.yoctoproject.org/meta-security |
 | meta-rauc| Layer controlling and performing secure software updates for embedded Linux | https://github.com/rauc/meta-rauc |
-| poky | Build tool and metadata included in a reference distribution | https://git.yoctoproject.org/poky |
+| meta-chargebyte | BSP layer for EVAcharge SE, Tarragon & Charge SOM | https://github.com/chargebyte/meta-chargebyte |
+| meta-chargebyte-distro | Distribution adaptations layer | https://github.com/chargebyte/meta-chargebyte-distro |
 
 This layering approach increases flexibility to expand your project. You can add layers, which in turn would add packages essential for the distribution you want to build. Layers are usually available as repositories. Information on how to include or remove layers will be given in [Section 3.3](#addorremove). Note that no charging capabilities will be included in the Linux distribution created by this setup.
```

Comment:
I think this needs to be adapted, right? If Charge SOM will be solely flashed with EVerest, then we do have charging capabilities. This point makes me question if this branch should have been based on `kirkstone-everest` and not `kirkstone`.

Reply:
No, I still want to keep the current approach of having a BSP-only layer without any charging stack or whatever. When this branch is merged, then we merge it into kirkstone-everest. At this point, the adaptions for the readme in this branch are made.